README.md

```diff
@@ -1 +1,68 @@
-# conda-recipes
+# Conda Recipes
+[![Travis Build Status](https://travis-ci.org/RNAer/conda-recipes.png?branch=master)](https://travis-ci.org/RNAer/conda-recipes)
```

Comment:
forgot to change that.

On Mon, Oct 19, 2015 at 1:42 PM, Jorge Cañardo Alastuey <
notifications@github.com> wrote:

> In README.md
> <https://github.com/biocore/conda-recipes/pull/5#discussion_r42423129>:
>
> > @@ -1 +1,68 @@
> > -# conda-recipes
> > +# Conda Recipes
> > +[![Travis Build Status](https://travis-ci.org/RNAer/conda-recipes.png?branch=master)](https://travis-ci.org/RNAer/conda-recipes)
>
> This should be biocore's travis-ci instead of RNAer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/conda-recipes/pull/5/files#r42423129>.
>